server/src/calls.ts

```diff
@@ -191,7 +191,7 @@ async function findNonDefinitionReferences(tspClient: TspClient, args: tsp.FileS
     return (await findReferences(tspClient, args)).filter(ref => !ref.isDefinition);
 }
 
-async function findReferences(tspClient: TspClient, args: tsp.FileSpan): Promise<tsp.ReferencesResponseItem[]> {
+async function findReferences(tspClient: TspClient, args: tsp.FileSpan): Promise<readonly tsp.ReferencesResponseItem[]> {
```

Comment:
Do you know if there's a difference between these two versions?

```suggestion
async function findReferences(tspClient: TspClient, args: tsp.FileSpan): Promise<ReadonlyArray<tsp.ReferencesResponseItem>> {
```

Reply:
Looks like there's not much of a practical difference as of TS 3.4 https://stackoverflow.com/questions/48300869/difference-between-readonly-and-readonlyarray#48301181